device-types/Avocent/ACS8032.yaml

```diff
@@ -0,0 +1,89 @@
+---
+manufacturer: Avocent
+model: ACS 8032
+slug: acs8032
+part_number: ACS-8032
+u_height: 1
+is_full_depth: false
```

Comment:
Please add a comment in markdown format if you have a link to the manufacturer provided datasheet

Reply:
There are several models of type 8032, so I refined the definition of one of the model + added link to the description